cypress/integration/plugins/index-management-dashboards-plugin/aliases.js

```diff
@@ -26,11 +26,25 @@ describe("Aliases", () => {
   });
 
   beforeEach(() => {
+    // Intercept the specific POST request
+    cy.intercept("POST", "/api/ism/apiCaller", (req) => {
+      if (req.body.data && req.body.data.name === "**" && req.body.data.s === "alias:desc" && req.body.endpoint === "cat.aliases") {
+        req.alias = "apiCaller"; // Assign an alias directly if the condition is met
+      }
+    });
+
     // Visit ISM OSD
     cy.visit(`${BASE_PATH}/app/${IM_PLUGIN_NAME}#/aliases`);
 
-    // Common text to wait for to confirm page loaded, give up to 60 seconds for initial load
-    cy.contains("Rows per page", { timeout: 60000 });
+    // Wait for 120 seconds for OSD to start.
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(120000);
```

Comment:
Don't want this to slow down the testing, I am looking for the way to make sure OSD starts, 2 things worth try

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/f6ce6e707035f791ed98a9edeab785d0785a8dbd/.github/workflows/cypress_workflow.yml#L149
https://github.com/opensearch-project/index-management-dashboards-plugin/blob/1c8f0c660a6a7ef4960862e67ab8140a8d156691/.github/workflows/cypress-workflow.yml#L75

For now you can just remove this wait, and we can try these 2 things in another PR

Reply:
@bowenlan-amzn Done! Could you please create the appropriate issue for this and assign it to me? I will get started on it in a separate PR.

Reply:
Create one https://github.com/opensearch-project/index-management-dashboards-plugin/issues/938 but still need your comment to assign, thanks!